tracing/ops_tracing/_buffer.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Buffer for tracing data in the ops-tracing extension."""
+
+from __future__ import annotations
+
+import contextlib
+import functools
+import pathlib
+import sqlite3
+from typing import TYPE_CHECKING, Callable
+
+from ._const import (
+    BUFFER_SIZE,
+    DB_RETRY,
+    DB_TIMEOUT,
+    DEFAULT_PRIORITY,
+    LONG_DB_TIMEOUT,
+    OBSERVED_PRIORITY,
```

Comment:
I find `const.py` style files just become a mish-mash of concerns. What about just defining these constants here, or in the files that use them, and remove `_const.py`?